### PR TITLE
NOTICK - added intro for CENM 1.5.6 release notes

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/1.5/cenm/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ title: Release notes
 
 ## Corda Enterprise Network Manager 1.5.6
 
+CENM 1.5.6 fixes two vulnerability issues:
```
Comment (colin-hillis) [inline content/en/platform/corda/1.5/cenm/release-notes.md:19]:
```suggestion
CENM 1.5.6 fixes two vulnerabilities by:
```

File: content/en/platform/corda/1.5/cenm/release-notes.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ title: Release notes
...
+
 * Due to a vulnerability, Jackson Databind is now excluded.
```
Comment (colin-hillis) [inline content/en/platform/corda/1.5/cenm/release-notes.md:21]:
```suggestion
* excluding Jackson Databind.
```

File: content/en/platform/corda/1.5/cenm/release-notes.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ title: Release notes
...
 * Due to a vulnerability, Spring Framework is now excluded.
```
Comment (colin-hillis) [inline content/en/platform/corda/1.5/cenm/release-notes.md:22]:
```suggestion
* excluding Spring Framework.
```

**Review by colin-hillis — CHANGES_REQUESTED** [review]
Something along these lines might read better.

File: content/en/platform/corda/1.5/cenm/release-notes.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/1.5/cenm/release-notes.md:21]: Done

File: content/en/platform/corda/1.5/cenm/release-notes.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/1.5/cenm/release-notes.md:19]: Done

File: content/en/platform/corda/1.5/cenm/release-notes.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/1.5/cenm/release-notes.md:22]: Done.

**Review by yarivus — COMMENTED** [review]
thx @paulmoloneyr3

**Pull request merged**